src/include/duckdb/catalog/default/builtin_types/types.hpp

```diff
@@ -0,0 +1,94 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/catalog/default/builtin_types/types.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+// This file is generated by scripts/generate_builtin_types.py
+
+#pragma once
+
+#include "duckdb/common/types.hpp"
+#include "duckdb/common/case_insensitive_map.hpp"
+
+namespace duckdb {
+
+const case_insensitive_map_t<LogicalTypeId> &GetInternalTypes() {
+	static const std::pair<std::string, LogicalTypeId> types[] = {
```

Comment:
Can this use the `DefaultType` struct that was used previously?

It likely doesn't really matter but this is keeping a lot more data and more complex structures with destructors statically allocated than before.

Reply:
I turned it into a `std::array` instead now